### ISSUE TITLE
Incompatible with 1.20.6

### ISSUE THREAD

**Issue by RoonMoonlight** [Lortseam/ugly-scoreboard-fix#11]
Hello, it seems the Ugly Scoreboard Fix causing the crash with 1.20.4 version, can you fix the bug to make it compatible with 1.20.4?

Link to the log:
https://mclo.gs/QkctR6x

Thanks

**Comment by Lortseam** [comment]
Yes, I have not updated this mod for 1.20.3/4 yet.

**Comment by Obscure2020** [comment]
Any timeline on when you might do so?

**Comment by Obscure2020** [comment]
Hey... any news here??

**Comment by RoonMoonlight** [comment]
The last commit was 2 months ago so this mod is considered dead... This issue has been present for almost 2 months and we got nothing while 1.20.5 update is about to arrive.

I was using this as part of modpack but now guess I should consider using other mod that does the same.

**Comment by Obscure2020** [comment]
If you find a good replacement mod, let me know where I might find it. I'm not sunk without this mod, because I custom-roll my mod stack one by one rather than using a modpack, but this was a nice quality-of-life mod to have.

**Comment by RoonMoonlight** [comment]
I use this mod at the meantime until I can get better alternatives: https://modrinth.com/mod/scoreboard-helper

It does not work exactly like Ugly Scoreboard Fix but it at least provides basic customization features

Hope the author comes back to actively updated mod or someone makes fork of this mod to continue the development (Not sure it the author is able to check but try making pull request first if possible)

**Comment by Sotumney** [comment]
Likely dead, I really, really liked using this mod, but seeing that CompleteConfig won't be updated for 1.21, I think he's quit :(

**Comment by Lortseam** [comment]
> Likely dead, I really, really liked using this mod, but seeing that CompleteConfig won't be updated for 1.21, I think he's quit :(

No, I've not quit. The decision to stop the development of CompleteConfig has nothing to do with this mod.

Anyway, if someone wants to try, I'm open for pull requests to update this mod for 1.20.4. Otherwise, I can't say if/when I will update it myself.

**Comment by RoonMoonlight** [comment]
> No, I've not quit. The decision to stop the development of CompleteConfig has nothing to do with this mod.
> 
> Anyway, if someone wants to try, I'm open for pull requests to update this mod for 1.20.4. Otherwise, I can't say if/when I will update it myself.

Glad to hear you are still alive! Thank you for update

**Comment by venaminecraft** [comment]
> > Likely dead, I really, really liked using this mod, but seeing that CompleteConfig won't be updated for 1.21, I think he's quit :(
> 
> No, I've not quit. The decision to stop the development of CompleteConfig has nothing to do with this mod.
> 
> Anyway, if someone wants to try, I'm open for pull requests to update this mod for 1.20.4. Otherwise, I can't say if/when I will update it myself.

Thanks, looking forward to the 1.20.4 update 😊 
Take your time!!

**Comment by andreasdc** [comment]
I think only this mod is keeping me from updating to 1.20.4 :(

**Comment by Sotumney** [comment]
It's been 3 months, and still no new update :(

**Comment by RoonMoonlight** [comment]
I have noticed this mod seems haven't been updated for while, and I updated this issue to release this mod for 1.20.6
